NO-JIRA: update readme of OpenShift (with authentication) part (#14975)
openshift/console: README.md

oc extract secret/off-cluster-token -n openshift-console --to ./examples --confirm
```

> If you have ran `source ./contrib/oc-environment.sh` before then unset all env vars

Reviewer: ```suggestion
If you have ran `source ./contrib/oc-environment.sh` before, unset all environment variables by running the following command:
```

> If you have ran `source ./contrib/oc-environment.sh` before, unset all environment variables by running the following command:

Reviewer 2: so I would probably suggest to unset these at the begining of the `oc-environment.sh`

   - .ci-operator.yaml
   - Dockerfile.dev
   - Dockerfile.plugins.demo
     - For example, `tectonic-console-builder:27`
2. Update the dependencies in Dockerfile.builder file i.e. v18.0.0.
3. Run `./push-builder.sh` script build and push the updated builder image to quay.io.
   Note: You can test the image using `./builder-run.sh ./build-backend.sh`.

>    - To update the image on quay.io, you need edit permission to the quay.io/coreos/ tectonic-console-builder repo.

Reviewer: ```suggestion
   - To update the image on quay.io, you must have permission to edit the quay.io/coreos/ tectonic-console-builder repository.
```
